doc/src/manual/style-guide.md

```diff
@@ -123,7 +123,7 @@ Types such as `Union{Function,AbstractString}` are often a sign that some design
 When creating a type such as:
 
 ```julia
-type MyType
+struct MyType
```

Comment:
you could only instantiate a value with one of the union members, and not change it